### PR TITLE
Pull request for editing stagefile.py

### PR DESCRIPTION
**Pull request by emakono**
Hi team, could you please review my pull request? Thanks
Content : added one line in stagefile.py

### CODE REVIEW

**Comment by emakono** [comment]
Hi team, could you please review my pull request? Thanks
Content : added one line in stagefile.py

File: stagefile.py
Comment (Cheyenne077) [inline stagefile.py:1]: Great!
We`re making progress.

**Review by Cheyenne077 — APPROVED** [review]
Making progress, great job!

**Comment by Pride22** [comment]
congrates team

**Review by Pride22 — APPROVED** [review]
this is great group 4, we are getting there.

**Comment by lunaafrique** [comment]
nice

**Review by vickiemai — COMMENTED** [review]
all correct

**Pull request merged**